### PR TITLE
Get number of points along the blade

### PR DESCRIPTION
**Pull request by tonyinme**
## Pull request type

Please check the type of change introduced:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## Checklist

The following is included:

- [ ] new unit-test(s)
- [ ] new regression test(s)
- [ ] documentation for new capability

This PR was tested by running:

- the unit tests
  - [ ] on GPU 
  - [ ] on CPU 
- the regression tests
  - [ ] on GPU 
  - [ ] on CPU 

## Additional background

This was a former bugfix that was reintroduced. 
This is a re-implementation of the bug fix. 

[Issue](https://github.com/Exawind/amr-wind/issues/1038)

### CODE REVIEW

File: amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H [hunk 1]
```diff
@@ -243,7 +243,7 @@ struct InitDataOp<TurbineFast, SrcTrait>
         amrex::ParallelDescriptor::Comm_dup(
             amrex::ParallelDescriptor::Communicator(), tdata.tcomm);
 
-        int sz_info[4]{0, 0, 0, 0};
+        int sz_info[5]{0, 0, 0, 0, 0};
```
Comment (marchdf) [inline amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:246]: I would be inclined to make this an `amrex::Vector<int> sz_info(5, 0)`. Also, is there an assert you can add somewhere to check that it was correctly populated? Maybe below on line 255? I would hate for this to happen again.

Comment (marchdf) [reply amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:246]: Different type, like a pointer? You would get that with `sz_info.data()`. Then your assert would just be `sz_info.size() == 5`

Comment (tonyinme) [reply amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:246]: The problem was not in the assert. It is in another fast function where it gets called. 
This is the error
`/home/lmartine/amr-wind-tony/amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:264:52: error: no matching function for call to 'Bcast(amrex::Vector<int>&, int, const int&, MPI_Comm&)'
             sz_info, 3, info.root_proc, tdata.tcomm);`

We need to leave it as `int sz_info[5]{0, 0, 0, 0, 0};`

Comment (marchdf) [reply amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H:246]: Lets drop this, I have an upcoming PR that will fix all these.

**Comment by marchdf** [comment]
There is actually another bug here. I am moving these fixes to another PR and adding the other fix I found.

**Comment by marchdf** [comment]
@tonyinme can you check to make sure I moved the fixes right? It will be #1043 and it will be in `amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H`. You can also see what I mean by using `.begin()` to get around that type error you were seeing. And the bugfix I was talking about was the MPI broadcast had the wrong size for `sz_info`.

**Comment by tonyinme** [comment]
> @tonyinme can you check to make sure I moved the fixes right? It will be #1043 and it will be in `amr-wind/wind_energy/actuator/turbine/fast/turbine_fast_ops.H`. You can also see what I mean by using `.begin()` to get around that type error you were seeing. And the bugfix I was talking about was the MPI broadcast had the wrong size for `sz_info`.


Yes, I went over it and the implementation looks good.

**Pull request closed without merging**